packages/dashboard/package.json

```diff
@@ -1,182 +1,76 @@
 {
   "name": "rmf-dashboard",
   "version": "0.0.1",
+  "type": "module",
   "private": true,
   "scripts": {
-    "analyze": "source-map-explorer 'build/static/js/*.js'",
-    "build": "pnpm run --filter {.}^... build && node scripts/build.js",
+    "build": "vite build",
     "build-storybook": "storybook build",
-    "setup": "node ./scripts/setup/setup.js",
```

Comment:
Can you help update the documentation on what is the process of using custom resources with these changes?

We can also merge https://github.com/open-rmf/rmf-web/pull/970 first, where I've updated documentation too with more information about the various configurations, before you start making changes

Reply:
https://github.com/open-rmf/rmf-web/pull/977/commits/d8d2224de9b7ee59ec57d2027c96a73ca211dd2b